vividus-plugin-kafka/src/main/java/org/vividus/bdd/steps/kafka/KafkaSteps.java

```diff
@@ -58,4 +101,88 @@ public void sendData(String data, String topic) throws InterruptedException, Exe
     {
         kafkaTemplate.send(topic, data).get(WAIT_TIMEOUT_IN_MINUTES, TimeUnit.MINUTES);
     }
+
+    /**
+     * Starts the Kafka consumer with the provided configuration to listen the specified topics. The consumer must be
+     * stopped when it's not needed.
+     *
+     * @param topics the comma-separated set of topics to listen
+     */
+    @When("I start consuming messages from Kafka topics `$topics`")
+    public void startKafkaListener(Set<String> topics)
+    {
+        BlockingQueue<String> messageQueue = new LinkedBlockingDeque<>();
+        testContext.put(MESSAGES_KEY, messageQueue);
+        ContainerProperties containerProperties = new ContainerProperties(topics.toArray(new String[0]));
+        containerProperties.setMessageListener(
+                (MessageListener<String, String>) data -> messageQueue.add(data.value()));
+        KafkaMessageListenerContainer<String, String> container = new KafkaMessageListenerContainer<>(consumerFactory,
+                containerProperties);
+        container.start();
+        testContext.put(LISTENER_KEY, container);
+        LOGGER.info("Kafka message listener is started");
```

Comment:
does it make sense to add topics into the log message?

Reply:
the data is logged by kafka client:
<img width="847" alt="log" src="https://user-images.githubusercontent.com/5081226/100099931-1d23ce00-2e71-11eb-85b2-7eb4248fa789.png">